content/references/http-websocket-apis/public-api-methods/clio-methods/ledger-clio.md

```diff
@@ -189,25 +189,10 @@ The response follows the [standard format][], with a successful result containin
 | `ledger_hash`                  | String  | Unique identifying hash of the entire ledger. |
 | `ledger_index`                 | Number  | The [Ledger Index][] of this ledger. |
 | `validated`                    | Boolean | _(May be omitted)_ If `true`, this is a validated ledger version. If omitted or set to `false`, this ledger's data is not final. |
-| `queue_data`                   | Array   | _(Omitted unless requested with the `queue` parameter)_ Array of objects describing queued transactions, in the same order as the queue. If the request specified `expand` as true, members contain full representations of the transactions, in either JSON or binary depending on whether the request specified `binary` as true. Added by the [FeeEscalation amendment][]. [New in: rippled 0.70.0][] |
-| `diff`                         | Object   | _(Omitted unless requested with the `diff` parameter)_ Object containing an array of hashes that were added, modified, or deleted as part of applying transactions for the ledger.
+| `diff`                         | Object  | _(Omitted unless requested with the `diff` parameter)_ Object containing an array of hashes that were added, modified, or deleted as part of applying transactions for the ledger.
 
 The following fields are deprecated and may be removed without further notice: `accepted`, `hash` (use `ledger_hash` instead), `seqNum` (use `ledger_index` instead), `totalCoins` (use `total_coins` instead).
```

Comment:
I believe these fields have actually been removed, so this should be updated too.